blog/composer.json

```diff
@@ -59,14 +59,15 @@
         "yiisoft/input-http": "^1.0",
         "yiisoft/log": "^2.0",
         "yiisoft/log-target-file": "^3.0",
-        "yiisoft/mailer": "^5.0",
-        "yiisoft/mailer-symfony": "^3.0",
+        "rossaddison/mailer": "dev-master",
+        "rossaddison/mailer-symfony": "*",    
```

Comment:
Why?

Reply:
@rossaddison  you can link your packages locally with yiisoft/yii-dev-tool
Take a look at it

Reply:
I could not get the yiisoft/mailer to function at the time due to a compatability issue with yiisoft/view i think it was at the time. So used my temporary forks instead. I do not believe the blog/composer.json can be compiled at the moment with composer.